openpgp/errors/errors.go

```diff
@@ -9,6 +9,21 @@ import (
 	"strconv"
 )
 
+const (
+	// GenericParsingErrorMessage is a generic error message for parsing errors
+	// to reduce the risk of oracle attacks.
+	GenericParsingErrorMessage = "parsing error"
+)
+
+var (
```

Comment:
Can these be `const` as well?

Reply:
Structs cannot be constant in go: https://go.dev/ref/spec#Constants